src/main/java/com/artipie/api/ManageUsers.java

```diff
@@ -109,7 +119,40 @@ public void remove(final String uid) {
             );
             return;
         }
-        throw new IllegalStateException(String.format("User %s does not exist", uid));
+        throw new IllegalStateException(String.format("User %s does not exist", uname));
+    }
+
+    @Override
+    public void alterPassword(final String uname, final JsonObject info) {
+        if (this.users().map(yaml -> yaml.yamlMapping(uname) != null).orElse(false)) {
+            YamlMappingBuilder all = Yaml.createYamlMappingBuilder();
+            final YamlMapping users = this.users().get();
+            for (final YamlNode node : users.keys()) {
+                final String val = node.asScalar().value();
+                if (!uname.equals(val)) {
```

Comment:
Note no change required: It is just example of usage standard java style of coding, it is not requirement to modify code:
Line 131: for variable 'val' is good name is 'userName'
It is good example for usage CamelCase

Reply:
`name` is enough in this case, we work with users here, so it's obvious that `name` is the name of a user